README.md

```diff
@@ -65,58 +67,109 @@ This step should be done in the case that changes have been made to any of the y
 ```shell script
-operator-sdk bundle create $BUNDLE_REPOSITORY:$VERSION_TAG -d deploy/olm-catalog/windows-machine-config-operator/0.0.0 \
---channels alpha --default-channel alpha --image-builder podman
+operator-sdk bundle create $BUNDLE_REPOSITORY:$BUNDLE_TAG -d deploy/olm-catalog/windows-machine-config-operator/manifests \
+--channels alpha --default-channel alpha
 ```
 The variables in the command should be changed to match the container image repository you wish to store the bundle in.
 You can also change the channels based on the release status of the operator.
+This command should create a new bundle image. Bundle image and operator image are two different images. 
 
-You should then push the image to the remote repository
+You should then push the newly created bundle image to the remote repository
 ```shell script
-podman push $BUNDLE_REPOSITORY:$BUNDLE_VERSION
+podman push $BUNDLE_REPOSITORY:$BUNDLE_TAG
 ```
 
 You should verify that the new bundle is valid:
 ```shell script
-operator-sdk bundle validate $BUNDLE_REPOSITORY:$BUNDLE_VERSION --image-builder podman
+operator-sdk bundle validate $BUNDLE_REPOSITORY:$BUNDLE_TAG
 ```
 
 ### Creating a new operator index
 An operator index is a collection of bundles. Creating one is required if you wish to deploy your operator on your own
 cluster.
 
 ```shell script
-opm index add --bundles $BUNDLE_REPOSITORY:$VERSION_TAG --tag $INDEX_VERSION
+opm index add --bundles $BUNDLE_REPOSITORY:$BUNDLE_TAG --tag $INDEX_REPOSITORY:$INDEX_TAG
+```
+
+You should then push the newly created index image to the remote repository
+```shell script
+podman push $INDEX_REPOSITORY:$INDEX_TAG
 ```
 
 #### Editing an existing operator index
 An existing operator index can have bundles added to it:
 ```shell script
-opm index add --from-index $INDEX_VERSION
+opm index add --from-index $INDEX_REPOSITORY:$INDEX_TAG
 ```
 and removed from it:
 ```shell script
-opm index rm --from-index $INDEX_VERSION
+opm index rm --from-index $INDEX_REPOSITORY:$INDEX_TAG
 ```
 
 ### Deploying the operator on a local cluster
 #### Openshift Console
 This deployment method is currently not supported. Please use the [CLI](#cli)
 
+#### Create Secrets
+Create the cloud provider and cloud private key secrets. The cloud-private-key should match the keypair used in the
+Windows Machine Config CR you will use.
+```shell script
+# Change paths as necessary
+oc create secret generic cloud-credentials --from-file=credentials=/home/$USER/.aws/credentials
+oc create secret generic cloud-private-key --from-file=private-key.pem=/home/$USER/.ssh/$keyname
+```
+
+Put the kubeconfig you wish to use in a secret. In order to use the permissions of the windows-machine-config-operator
+service account the kubeconfig should be generated from the service account.
+```shell script
+# Change paths as necessary
+oc create secret generic kubeconfig --from-file=kubeconfig=/path/to/kubeconfig
+```
+
+#### Running locally with OLM 
```

Comment:
Doesn't this imply that you don't need to create the bundle image? If so you should call it out.
`Running locally with OLM`
with subsections
`Without bundle and index images`
`With bundle and index images`